fuel_test/localrc.default

```diff
@@ -10,6 +9,10 @@ export FUEL_TEST_LOG_DIR=${FUEL_TEST_LOG_DIR:-"fuel_test_logs"}
 export FM_SNAPSHOT="fuel_ci"
 export ENV_NAME="TestEnv"
 
+export BRANCH=${ZUUL_BRANCH:-"master"}
+export MERGE_REPO_URL=${MERGE_REPO_URL:-"https://review.openstack.org/p"}
```

Comment:
The two defaults of 'master' and 'review.openstack.org/p' don't make sense together